airflow/kubernetes/pod_generator.py

```diff
@@ -445,29 +455,27 @@ def make_unique_pod_id(pod_id: str) -> str | None:
         r"""
         Kubernetes pod names must consist of one or more lowercase
         rfc1035/rfc1123 labels separated by '.' with a maximum length of 253
-        characters. Each label has a maximum length of 63 characters.
+        characters.
 
         Name must pass the following regex for validation
         ``^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$``
 
+        (But note that this method actually doesn't guarantee that!)
```

Comment:
Mostly for my own learning: Is this asserted somewhere else further up or down the stack?
If not, should it be asserted here or elsewhere?

Reply:
it's not really asserted.   But it follows from a consequence of using slugify in `create_pod_id`.  And the behavior of that function is tested.

This one just adds a random suffix.  So it assumes whatever it's given is already valid.

Reply:
(might be better just to chop that regex comment)